FAQs.md

```diff
@@ -0,0 +1,50 @@
+# FAQ
+
+## What is kubemci?
+
+kubemci is a command line tool that allows users to create multicluster ingresses
+using [Google Cloud Load Balancer (GCLB)](https://cloud.google.com/load-balancing/)
+It enables users to perform [cross-region load
+balancing](https://cloud.google.com/compute/docs/load-balancing/http/cross-region-example) using
+multiple kubernetes clusters in different Google Cloud regions.
+
+## Does it work with other load balancer providers
+
+It works only with Google Cloud Load Balancer for now. In future, we can
```

Comment:
I think the answer needs to be two part.
1. It programs the GCLB and the many resources that are required to enable functional ingress across multiple clusters.
2. It sets up geo-awareness that is provided in GCLB through anycast. Porting it to other load balancers will require consideration of anycast or geo-dns to provide similar functionality.